src/disjoint_set.jl

```diff
@@ -182,6 +182,8 @@ Base.empty(s::DisjointSet{T}, ::Type{U}=T) where {T,U} = DisjointSet{U}()
 function Base.sizehint!(s::DisjointSet, n::Integer)
     sizehint!(s.intmap, n)
     sizehint!(s.revmap, n)
+    sizehint!(s.internal.parents, n)
+    sizehint!(s.internal.ranks, n)
```

Comment:
Can't we just do:
```suggestion
    sizehint!(s.internal, n)
```

Reply:
We can't because `sizehint!` is not defined for a `IntDisjointSet`.

Reply:
We should define it, and then do this?